apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.e2e.ts

```diff
@@ -15,6 +15,12 @@ describe('Get My Environments - /environments (GET)', async () => {
     expect(body.data.length).to.be.greaterThanOrEqual(2);
     for (const elem of body.data) {
       expect(elem._organizationId).to.eq(session.organization._id);
+
+      if (elem._id !== session.environment._id) {
```

Comment:
@djabarovgeorge I added two assertions in test for use case. I think it's enough as test itself is simple.